ci/gitlab-cscs.yml.only-ault

```diff
@@ -0,0 +1,11 @@
+stages:
```

Comment:
Do we still need this file?

Reply:
no, we don't. That was a backup while I was debugging the Ault pipeline.